js/deck-gl/utils/tiles.js

```diff
@@ -28,14 +28,21 @@ export const create_get_tile_data = (
     const { x, y, z } = index;
     const full_url = `${base_url}/pyramid_images/${image_name}_files/${max_image_zoom + z}/${x}_${y}${image_format}`;
 
-    // If AWS credentials are provided, use aws.fetch to sign the request
-    const fetch_fn = aws
-      ? (url, fetchOptions) => aws.fetch(url, fetchOptions)
-      : undefined;
+    // Separate fetch options so we can merge them with loader options
+    const { fetch: fetchOptions = {}, ...loaderOptions } = options;
+
+    // Fetch function that preserves headers and supports aws signed requests
+    const fetch_fn = async (url, fetchOpts = {}) => {
+      const merged = { ...fetchOptions, ...fetchOpts };
+      if (aws) {
+        return aws.fetch(url, merged);
+      }
+      return fetch(url, merged);
+    };
```

Comment:
Does this fix `fetch` error when visualizing Visium-HD data in Celldega?

Reply:
No, this doesn't fix that. 

I'm thinking we will want to take the approach of visualizing Visium HD data using landscape_ist for now since the newer versions have segmented cells/transcripts. But we can discuss this in another pull request.